app/controllers/clubs_controller.rb

```diff
@@ -0,0 +1,13 @@
+class ClubsController < ApplicationController
+
+  def show
+    club_id = params[:id]
+    @club = ClubFacade.get_club(club_id)
+    @club_users = ClubFacade.get_club_users(club_id)
+    # @book_id = @club.book_id
+    # @book = BookFacade.book_show(@book_id)
+    # need to update book ids in seed to match real books 
```

Comment:
My thought was that when a book was chosen for the book club, it would some how be able to pass its id over right?

Reply:
I understand. So when the club is created it will fill the book_id?